src/main/java/org/broadinstitute/hellbender/tools/funcotator/mafOutput/MafOutputRenderer.java

```diff
@@ -148,6 +148,9 @@
     /** Fields that should be removed in the final MAF file. */
     private final Set<String> excludedOutputFields;
 
+    /** The version of the tool used to produce the MAF file. */
```

Comment:
Might as well put this as a field in the `OutputRenderer` parent class itself. I can't imagine ever not needing it.

Reply:
Done